core/src/main/kotlin/endpoints/plugin/PluginDispatcher.kt

```diff
@@ -324,6 +324,9 @@ internal class PluginDispatcher(override val kodein: Kodein) : KodeinAware {
             contentType = ContentType.MultiPart.Any,
             status = statusCode
         )
+        is FileResponse -> call.respondFile(
+            file = response.data
```

Comment:
can we remove a temp file after sending it?

Reply:
yes, we can not to send file but outputSteam , see new commit